notebooks/api/0.8/10-container-images.ipynb

```diff
@@ -1277,37 +1278,47 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# syft absolute\n",
-    "from syft.custom_worker.builder import CustomWorkerBuilder\n",
-    "from syft.service.response import SyftError\n",
-    "from syft.service.response import SyftSuccess\n",
-    "\n",
+    "# Since the containers are delete, we should be able to delete the image\n",
+    "assert isinstance(delete_res, sy.SyftSuccess), str(delete_res)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "104",
+   "metadata": {},
+   "source": [
+    "##### Request to build the image and create the pool at the same time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "105",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "custom_dockerfile_str_3 = f\"\"\"\n",
+    "FROM openmined/grid-backend:{syft_base_worker_tag}\n",
     "\n",
-    "def test_image_build(config: str, tag: str, pull: bool, **kwargs):\n",
-    "    builder = CustomWorkerBuilder()\n",
-    "    try:\n",
-    "        result = builder.build_image(\n",
-    "            config=config, tag=tag, pull=pull, rm=True, forcerm=True, **kwargs\n",
-    "        )\n",
-    "        return SyftSuccess(message=result.logs)\n",
-    "    except Exception as e:\n",
-    "        return SyftError(message=f\"Failed to build image !! Error: {str(e)}.\")\n",
+    "RUN pip install recordlinkage\n",
+    "\"\"\".strip()\n",
     "\n",
```

Comment:
A Quick question, many a times, this tests flakes , due to the fact , that, it was not able to delete a worker with a given UID.

were we able to fix that?

Reply:
i have not been able to repro this after this fix - it was likely an out of resource error.